tools/run_unit_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
There is no formal rule in place, but it seems that we're putting CI (and testing is CI) related scripts in `.circleci/`. `tools/` is more user oriented.

This file literally depends on `.circleci/test-targets.txt`.